schema-registry/serdes/src/main/java/com/hortonworks/registries/schemaregistry/serdes/avro/AbstractAvroSerDesProtocolHandler.java

```diff
@@ -70,18 +71,17 @@ public Byte getProtocolId() {
     }
 
     @Override
-    public void handlePayloadSerialization(OutputStream outputStream, Object input) {
-        avroSerDesHandler.handlePayloadSerialization(outputStream, input);
+    public void handlePayloadSerialization(OutputStream outputStream, Object input, DatumWriterProvider datumWriterProvider) {
```

Comment:
This is breaking backward compatible API. Can we maintain the existing method which internally uses default implementation to invoke this newly added method?